compute_endpoint/tests/unit/test_identity_mapper.py

```diff
@@ -30,6 +30,16 @@ def _external_identity_mapper_conf() -> str:
     }]""".strip()
 
 
+ab_identity_set = [{"uname": "a@a.local"}, {"uname": "b@b.local"}]
+ba_identity_set = [{"uname": "b@a.local"}, {"uname": "a@a.local"}]
+cb_identity_set = [{"uname": "a@c.local"}, {"uname": "b@b.local"}]
+
```

Comment:
These are all 2-entry sets.  Perhaps a bit more variety to be safe?  ie. single entry, > 2?